accelerator/http.c

```diff
@@ -177,6 +177,18 @@ static inline int process_send_trytes_request(ta_http_t *const http, char const
   return set_response_content(ret, out);
 }
 
+static inline int process_get_ta_info_request(ta_http_t *const http, char **const out) {
+  status_t ret = SC_OK;
```

Comment:
Is it necessary to assign SC_OK to ret？

Reply:
I think we doesn't need to assign it there.
How about open another PR for refactoring these?

Reply:
Remove this assignment.

Reply:
It does not effect since `ret` is just a local variable.